app/controllers/approvals_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
 
     current_student.add(@approvable)
 
-    @graduated = current_student.total_credits >= 450 && previous_total_credits < 450
+    @graduated = current_student.total_credits >= 450 && previous_total_credits < 450 && groups_credits_met?
```

Comment:
should this be `current_student. groups_credits_met?`

Reply:
Updated in 6c77365733c372e7f64a1d4d651f8af9e9164153